packages/components/src/styles/media/media.stories.ts

```diff
@@ -35,25 +39,21 @@ export const Default = {
   }
 };
 
-export const MediaSample = {
-  parameters: {
-    controls: {
-      disable: true
-    }
-  },
-  render: (args: any) => {
-    return generateTemplate({
-      options: {
-        templateContent: `
-          <figure class="%CLASSES% max-w-xl p-4">
-            <div class="sd-copyright" style="--copyright: '© 2024 Solid Design System';">
-              <img src="./placeholders/images/generic.jpg" alt="A generic placeholder jpg" class="aspect-video object-cover"/>
-            </div>
-            <figcaption>%SLOT%</figcaption>
-          </figure>
-      `
-      },
-      args
-    });
-  }
+/**
+ * Use the `sd-media--inverted` class when displayed on primary background.
```

Comment:
Use the ”inverted” class when displayed on primary background if description is added.

Reply:
We have this sentence for all other style comps... in Figma it is called "&--inverted"

How about this sentence:

Use the ”&--inverted” class when displayed on primary background if description is added.

Otherwise we need to specify figma and code only things and I think that’s not worth it ... a consolidated sentence that works on both ends.

Reply:
I will update it to all the styles components then